src/chocolatey.tests/infrastructure.app/commands/ChocolateyListCommandSpecs.cs

```diff
@@ -111,22 +111,6 @@ public void should_add_short_version_of_localonly_to_the_option_set()
                 optionSet.Contains("l").ShouldBeTrue();
             }
 
-            [NUnit.Framework.Theory]
-            [NUnit.Framework.TestCase("localonly")]
-            [NUnit.Framework.TestCase("source")]
-            [NUnit.Framework.TestCase("user")]
-            [NUnit.Framework.TestCase("password")]
-            [NUnit.Framework.TestCase("cert")]
-            [NUnit.Framework.TestCase("certpassword")]
-            [NUnit.Framework.TestCase("approved-only")]
-            [NUnit.Framework.TestCase("download-cache-only")]
-            [NUnit.Framework.TestCase("disable-package-repository-optimizations")]
-            [Obsolete("Will be removed in v2.0.0 for the list command")]
-            public void should_add_deprecation_notice_to_option(string argument)
-            {
-                optionSet[argument].Description.ShouldContain("DEPRECATION NOTICE");
-            }
-

```

Comment:
This doesn't feel like it should be related to this PR? 🤔 In combination with other deprecations, absolutely, but on its own shouldn't this PR leave in the one deprecation message specifically about `choco list --local-only`?

Reply:
I had the same thought, but that felt like the wrong thing to do, given that we know that this work is coming out, and I was in that code anyway.  Looks like Kim is removing the same thing in his PR as well, so I am happy to rebase this work once his PR is merged.

Reply:
Going to merge this one, and rebase into Kim's since that's the order we're at right now...